priv/stops.json

```diff
@@ -9,10 +9,6 @@
     {"station": "G Government Center eastbound", "stop_ids": ["70201"]},
     {"station": "G Haymarket eastbound", "stop_ids": ["70203"]},
     {"station": "G North Station eastbound", "stop_ids": ["70205"]},
-    {"station": "G Science Park eastbound", "stop_ids": ["70207"]},
```

Comment:
Apologies for missing this, but while checking my work on this PR, I realized that I missed adding the new stops to these lists here. If I'm not mistaken, I think these are used for alerts so definitely important that they go out on Monday @losvedir @hannahpurcell

Reply:
Oh, yep, good catch, this file should be updated. This file is used to calculate which stations are adjacent to other stations, for the purposes of what to display on an alert. Namely, if a given station has a shuttle alert, along with all its neighbors, then we say that the station is closed. If one of its neighbors has service, then we consider the station an "endpoint" of the shuttle stretch, and we just blank the sign, expecting the PIO to put something useful on it about where or how to catch the shuttles.

(Alas, this would be better solved by an inference of an alert's activities, which specifies whether a given stop ID allows boarding, exiting, or riding, but that either didn't exist yet or didn't occur to us at the time we wrote this.)

In that regard, the stations being adjacent to each other in the list is important, and in fact we only use the list grouping names ("Green-E-eastbound") for organization; they don't really matter.

So I think the fix here would be to not delete these rows (even though technically the stations are not in the Green-Trunk), but to update Lechmere's stop IDs, and then add in Union Sq as two rows, between the two Lechmeres. (The list is kind of an outbound-then-inbound loop.)

Reply:
Gotcha, that makes sense. I was just checking out the unit test that's failing after adding them to the new stops and agreed that the fix is to change the IDs then add Union Sq in the middle.